.github/workflows/avm.ptn.app.container-job-toolkit.yml

```diff
@@ -0,0 +1,90 @@
+name: "avm.ptn.app.container-job-toolkit.yml"
+
+on:
+  schedule:
```

Comment:
This looks outdated, the trigger on schedule has been removed

Reply:
Thanks. I didn't find any template, so I copied from an existing action. Would you have the template link for me?